cmd/buildah/manifest.go

```diff
@@ -55,6 +55,7 @@ func init() {
 		manifestInspectDescription  = "\n  Display the contents of a manifest list or image index."
 		manifestPushDescription     = "\n  Pushes manifest lists and image indexes to registries."
 		manifestRmDescription       = "\n  Remove one or more manifest lists from local storage."
+		manifestExistsDescription   = "\n  If the manifest list exists in local storage, buildah manifest exists exits with 0, otherwise the exit code will be 1."
```

Comment:
Leave off the bit about exit status.  This should basically be the same as the `Short` description of the command, give or take punctuation and formatting.